cfgov/jinja2/v1/owning-a-home/explore-rates/index.html

```diff
@@ -93,23 +79,23 @@ <h4>Explore rate options</h4>
                                 <span class="a-range_labels-max">850</span>
                             </div>
                             <input id="credit-score"
-                                   class="a-range_input"
-                                   type="range"
-                                   min="600"
-                                   max="840"
-                                   step="20"
-                                   value="700">
+                                    class="a-range_input"
+                                    type="range"
+                                    min="600"
+                                    max="840"
+                                    step="20"
+                                    value="700">
                             <div id="slider-range" class="a-range_text"></div>
 
                             {#
-                              Credit alert for when low credit score is chosen.
+                                Credit alert for when low credit score is chosen.
                             #}
 
                             <div class="result-alert
                                         credit-alert
                                         u-hidden"
-                             id="credit-score-alert"
-                             role="alert">
+                                id="credit-score-alert"
+                                role="alert">
```

Comment:
This is one where the indentation change is actually improving the situation, but still not quite getting to the right attribute alignment.

Reply:
Fixed in https://github.com/cfpb/consumerfinance.gov/pull/6446/commits/d208ae3ed04e1ec25f3b948168d3197990a0ae75